setup.py

```diff
@@ -8,7 +8,7 @@
     "torchimagefilter@https://github.com/pmeier/torchimagefilter/archive/master.zip",
 )
 
-testing_requires = ("numpy", "pillow", "torchvision")
+testing_requires = ("requests", "pillow<7.0.0", "numpy", "torchvision")
```

Comment:
The current `torchvision==0.4.2` is not compatible with `pillow>=7.0.0`. This will be fixed in the next `torchvision` release and afterwards the `pillow` version restriction should be shifted to use `torchvision>0.4.2`.